app/views/schedules/_root_fields.erb

```diff
@@ -0,0 +1,16 @@
+<%= f.label :name %>
+<%= f.text_field :name, class: 'form-control' %>
+
+<h3>
+  Tasks
+</h3>
+<%= f.label :root_task_id, 'Root Task' %>
+<%= f.select :root_task_id,
+             Task.where(schedule_id: f.object.id).map { |t| ["##{t.id}", t.id] },
+             {include_blank: true, selected: (f.object.nil? || f.object.root_task.nil? ? '' : f.object.root_task.id)},
+             class: 'form-control' %>
+
+<div id="tasks" class="form-group">
+  <% # I want to just render the blank new Task I created in the controller, but I think this is failing because the schuled hasn't saved yet so it still thinks this is nil? %>
+    <%= render 'task_fields', f: f %>
```

Comment:
I noticed these errors too. It's what I was working on a week or so ago,
and I'm getting back to now. The seeds stuff is in the seeds branch though,
if you wanted to look at that real quick. I think I forgot to make a PR, so
I'll do that now.

On Tue, Mar 29, 2016 at 2:58 AM Kyle Oliveira <notifications@github.com>
wrote:

> In app/views/schedules/_root_fields.erb
> <https://github.com/Ohmbrewer/ohmbrewer/pull/89#discussion_r57679524>:
>
> > @@ -0,0 +1,16 @@
> > +<%= f.label :name %>
> > +<%= f.text_field :name, class: 'form-control' %>
> > +
> > +<h3>
> > +  Tasks
> > +</h3>
> > +<%= f.label :root_task_id, 'Root Task' %>
> > +<%= f.select :root_task_id,
> > +             Task.where(schedule_id: f.object.id).map { |t| ["##{t.id}", t.id] },
> > +             {include_blank: true, selected: (f.object.nil? || f.object.root_task.nil? ? '' : f.object.root_task.id)},
> > +             class: 'form-control' %>
> > +
> > +<div id="tasks" class="form-group">
> > +  <% # I want to just render the blank new Task I created in the controller, but I think this is failing because the schuled hasn't saved yet so it still thinks this is nil? %>
> > +    <%= render 'task_fields', f: f %>
>
> I'm getting an undefined method error when I try to create a new Schedule.
> It looks like the task_fields partial has a field for Type and Schedule
> doesn't have a Type. I think if you aren't calling it via Cocoon you might
> have to pass the root_task object directly into the partial.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ohmbrewer/ohmbrewer/pull/89/files/be4a8f019a91287b5306dbc14224c66a98eca0ab#r57679524>
>